core/connection_checker.js

```diff
@@ -108,12 +108,13 @@ Blockly.ConnectionChecker.prototype.getErrorMessage = function(errorCode,
       return 'Attempt to connect incompatible types.';
     case Blockly.Connection.REASON_TARGET_NULL:
       return 'Target connection is null.';
-    case Blockly.Connection.REASON_CHECKS_FAILED:
-      var connOne = /** @type {!Blockly.Connection} **/ (a);
-      var connTwo = /** @type {!Blockly.Connection} **/ (b);
-      var msg = 'Connection checks failed. ';
+    case Blockly.Connection.REASON_CHECKS_FAILED: {
```

Comment:
Just to clarify, this scope is to get rid of a lint error? Or is it functionally necessary? I see it's used later in this file, but I'm not really familiar with this syntax.

Reply:
Yeah, it's to get rid of a lint error. I believe the motivation is to avoid surprises/allow you to use the same variable name in multiple case statements, since without the braces those are the same scope.